Dockerfile

```diff
@@ -44,6 +45,15 @@ RUN curl -L "http://downloads.sourceforge.net/project/osslsigncode/osslsigncode/
 WORKDIR osslsigncode-1.7.1
 RUN ./configure && make && make install
 
+# build stubbed signtool.exe which can run under wine and call osslsigntool
+COPY ./main.go /gosigntool/main.go
+COPY ./osslsign.sh /gosigntool/osslsign.sh
```

Comment:
Can we just copy `osslsign.sh` straight to `/usr/local/bin/` instead of putting it in this temporary location? Also, I think if you `chmod +x` it in the git repo, then it will keep the executable bit when copied into the docker container, and you don't need to `chmod +x` it in the dockerfile.